resident/resident-service/src/main/java/io/mosip/resident/service/impl/IdentityServiceImpl.java

```diff
@@ -217,16 +218,18 @@ public Map<String, Object> getIdentityAttributes(String id, boolean includeUin,S
 				throw new ResidentServiceCheckedException(ResidentErrorCode.API_RESOURCE_ACCESS_EXCEPTION.getErrorCode(),
 						responseWrapper.getErrors().get(0).getErrorCode() + " --> " + responseWrapper.getErrors().get(0).getMessage());
 			}
-			Map<String, ?> identityResponse = new LinkedHashMap<>((Map<String, Object>) responseWrapper.getResponse());
-			Map<String, ?> identity = (Map<String, ?>) identityResponse.get(IDENTITY);
-
+			Map<String, Object> identityResponse = new LinkedHashMap<>((Map<String, Object>) responseWrapper.getResponse());
```

Comment:
Please add perpatual VID only if it is required in ui schema.

Reply:
changes done